lib/block-template-utils.php

```diff
@@ -98,6 +98,34 @@ function gutenberg_generate_block_templates_export_file() {
 	// Convert to a string.
 	$theme_json_encoded = wp_json_encode( $theme_json_raw, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE );
 
+	// Find any uploaded files.
+	$uploads = wp_upload_dir();
+	$pattern = '/"(' . preg_quote( $uploads['baseurl'], '/' ) . '\/[^"]+)"/';
+	preg_match_all( $pattern, $theme_json_encoded, $matches );
+
+	if ( ! empty( $matches ) ) {
+		$replacement     = '"file:./uploads/$1"';
```

Comment:
`/uploads` doesn't mean much in the context of a bundled theme. Let's dump theme somewhere else, e.g., `/assets/*`